addon/components/tags-widget/component.js

```diff
@@ -49,22 +49,28 @@ export default Ember.Component.extend({
                     this.send('removeATag', tag);
                 }
             });
-            // Populate the widget with the default tags passed in
-            let tags = this.get('tags');
-            this.$().importTags(tags.join(', '));
         });
     }),
 
+    didRender() {
+        // Rerender the list of tags whenever the node model changes. Useful if node.tags is not defined when page loads.
+        // Provide a default value in case tags weren't defined when component first rendered
+        this.$().importTags('');
```

Comment:
As for consolidation, it looks like the reset (line 58) can indeed be consolidated with populating the new tags (line 60). I'll push that with any other changes requested when review is done.

https://github.com/xoxco/jQuery-Tags-Input/blob/ae31b175fbfd0a0476822182ef52e76c2629e9c3/src/jquery.tagsinput.js#L171